src/main/java/at/hannibal2/skyhanni/features/commands/PartyCommands.kt

```diff
@@ -69,7 +75,7 @@ object PartyCommands {
     }
 
     fun customTabComplete(command: String): List<String>? {
-        if (command == "pk" || command == "pt" || command == "pp" && config.shortCommands) {
+        if (command == "pk" || command == "pt" || command == "pp" || command == "pd" && config.shortCommands) {
```

Comment:
This line should not be changed as you don't tab complete player names for p disband